Source/quests.cpp

```diff
@@ -123,7 +123,7 @@ void DrawButcher()
 	DRLG_RectTrans(x + 3, y + 3, x + 10, y + 10);
 }
 
-void DrawSkelKing(int q, int x, int y)
+void DrawSkelKing(quest_id q, int x, int y)

```

Comment:
This could take a QuestStruct&, but it is also only ever called for Q_SKELKING. Could even make this constexpr given how short the function is.